processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -806,7 +781,7 @@ public JavaExpressionMappingBuilder javaExpression(String javaExpression) {
         }
 
         public PropertyMapping build() {
-            Assignment assignment = new SourceRHS( javaExpression, null, existingVariableNames );
+            Assignment assignment = new SourceRHS( javaExpression, null, existingVariableNames, "" );
```

Comment:
You can leave the constructor. If it is there one needs to think about why it is there and it won't be forgotten. Empty is OK for me